docs/chat_managers/botpress_v12.md

```diff
@@ -6,6 +6,29 @@ You can use the AAQ endpoints through various chat managers. Below is an example
 
 ## Installation
 
+### Via Docker-Compose (behind NGINX with HTTPS)
+
+**Step 1:** Copy `deployment_botpress/template.env.nginx` to `deployment_botpress/.env.nginx` and edit it to set the variables
+
+    DOMAIN=
+
+!!! note "Note: The environment variables for botpress and its postgres db are currently hardcoded into `docker-compose.yml`."
```

Comment:
Docker compose use .env files by default if I remember correctly. Let's use them and no hard code it.